cloud/aws/aws_file.h

```diff
@@ -41,27 +41,6 @@
 #include <aws/s3/model/PutObjectResult.h>
 #include <aws/s3/model/ServerSideEncryption.h>
 
-// A few local defintions
-namespace {
-
-inline std::string GetBucket(const std::string& bucket_prefix) {
-  return "rockset." + bucket_prefix;
-}
-
-inline std::string GetStreamName(const std::string& bucket_prefix) {
-  return "rockset." + bucket_prefix;
-}
-inline Aws::String GetAwsBucket(const std::string& bucket_prefix) {
-  const std::string dd = GetBucket(bucket_prefix);
-  return Aws::String(dd.c_str(), dd.size());
-}
-inline Aws::String GetAwsStreamName(const std::string& bucket_prefix) {
-  const std::string dd = GetStreamName(bucket_prefix);
-  return Aws::String(dd.c_str(), dd.size());
-}
-
-}  // namespace
-
```

Comment:
The one problem with removing this is that now there are a lot of lines like `Aws::String(X.c_str(), X.size())` that you've introduced which are a bit clunky. Maybe add a helper function here like `AWS::String toAwsString(const std::string& s);` to clean it up